locales/en/plugin__odf-console.json

```
@@ -1153,6 +1153,25 @@
   "Share with presigned URL": "Share with presigned URL",
   "No objects found": "No objects found",
   "You do not have any objects in the bucket": "You do not have any objects in the bucket",
+  "Add Objects": "Add Objects",
+  "Drag and drop files/folders here.": "Drag and drop files/folders here.",
+  "Standard uploads have a size limit of up to 5GB in S3. For objects, multipart upload will upload the object in parts, which are assembled in the bucket.": "Standard uploads have a size limit of up to 5GB in S3. For objects, multipart upload will upload the object in parts, which are assembled in the bucket.",
+  "Upload": "Upload",
+  "Uploads": "Uploads",
+  "{{uploadedFiles}} of {{totalFiles}} files uploaded&nbsp;": "{{uploadedFiles}} of {{totalFiles}} files uploaded&nbsp;",
+  "Succeeded: {{uploadedFiles}}": "Succeeded: {{uploadedFiles}}",
+  "Failed files: {{failedFiles}}": "Failed files: {{failedFiles}}",
+  "Completion time: {{totalTimeElasped}}": "Completion time: {{totalTimeElasped}}",
+  "Total Remaining: {{totalReamining}}": "Total Remaining: {{totalReamining}}",
+  "Estimated time remaining: {{timeRemaining}}": "Estimated time remaining: {{timeRemaining}}",
+  "Transfer rate: {{uploadSpeed}}": "Transfer rate: {{uploadSpeed}}",
+  "Standard uploads have a size limit of up to 5 GB in S3. For large objects, something or the other will happen. We need to improve this alert. Or not show this. Doesnt make sense to repeat your messages in UI.": "Standard uploads have a size limit of up to 5 GB in S3. For large objects, something or the other will happen. We need to improve this alert. Or not show this. Doesnt make sense to repeat your messages in UI.",
```

same as https://github.com/red-hat-storage/odf-console/pull/1635/files#r1795747673
```suggestion
  "For large objects, something or the other will happen. We need to improve this alert. Or not show this. Doesnt make sense to repeat your messages in UI.": "For large objects, something or the other will happen. We need to improve this alert. Or not show this. Doesnt make sense to repeat your messages in UI.",
```

---

This is currently as it stands in the mockups. Let's update the text when the microcopy is ready.

---

5GB is more of a technical detail, microcopy team wouldn't know about it unless we tell them specifically... it's safer to remove than displaying incorrect info :(